pkg/metrics/metrics.go

```diff
@@ -34,6 +34,7 @@ const (
 	// KafkaRequestsStatus - kafka requests status metric
 	KafkaRequestsStatusSinceCreated = "kafka_requests_status_since_created_in_seconds"
 	KafkaRequestsStatusCount        = "kafka_requests_status_count"
+	KafkaRequestsStatusSinceLastSet = "kafka_requests_status_since_last_set"
```

Comment:
I wonder if the name still makes sense because we no longer aim to return the timestamp as a value for this metric. This now gives us information of the current status of a Kafka instance. Maybe something like `kafka_request_current_status_info`?